### DISCUSSION TITLE
1주일 간의 목표치

### DISCUSSION THREAD

**Discussion by sds1vrk** [prgrms-web-devcourse/slam-server#132] [General]
한주간의 백엔드 목표치를 적어보아요!

**Comment by sds1vrk** [comment]
## [22-02-07 ~ 22-02-13] 목표치 정하기 및 마지막 날에 진행 % 적기
ex) 헤이 - API 명세서 다 수정 및 테스트 코드 작성 (70%)

**Reply by KwonYeKyeong** [reply]
젤리(권예경)
- [ ] 관리자 도메인 API 수정 (33 / 100)
   - 농구장 등록 승인 API ✔️
   - 농구장 등록 거절 API 🔺
   - newCourt 목록 조회 API ❌

**Reply by locodingve** [reply]
1. 큰 목표 
- [ ] 세부목표 [현재잰행상황/목표치]
*보통 목표치는 100을 기준으로 합니다. 

---

1. notification 도메인 수정 (22-02-07 ~ 22-02-11, 평일)
- [ ] 유닛 테스트 (테스트 코드 짜서 확인) [ 10/100 ]
- [ ] 통합 테스트 [ 30/100 ]

2. 새로운 aws에 서버 이전하기 (22-02-12 ~ 22-02-13, 주말)
- [ ] CICD 구축 테스트 [ 60/100 ] 🙌
- [ ] (CICD 구축하기 전에) 우리 레포 옮는 것 도움 요청하기 [ 50/100 ] 🙌
- [ ] CICD 구축 하기(상황에 따라 다음주 업무가 될 수 있음) [ 30/100 ] 🙌
-  api 테스트 확인 (다음주 업무)
-  SSL 적용(다음주 업무)
-  통합테스트 (다음주 업무)

cf. 🙌 표시는 당일 업데이트 된 부분입니다.

**Comment by locodingve** [comment]
# [22-02-14 ~ 22-02-20] 목표치

**Reply by locodingve** [reply]
1. notification 도메인 수정 (22-02-07 ~ 22-02-11, 평일)
- [ ] 유닛 테스트 (테스트 코드 짜서 확인) [ 10/100 ]
- [ ] 통합 테스트 [ 30/100 ]

2. 새로운 aws에 서버 이전하기 (22-02-12 ~ 22-02-13, 주말)
- [ ] CICD 구축 테스트 [ 60/100 ] 
- [ ] (CICD 구축하기 전에) 우리 레포 옮는 것 도움 요청하기 [ 50/100 ] 
- [ ] CICD 구축 하기(상황에 따라 다음주 업무가 될 수 있음) [ 30/100 ] 
-  api 테스트 확인 (다음주 업무)
-  SSL 적용(다음주 업무)
-  통합테스트 (다음주 업무)

현재, 유닛테스트에서 이슈가 생겨서 진행이 더뎌지고 있습니다.

**Reply by KwonYeKyeong** [reply]
업무의 우선순위 순으로 적어보았습니다.

1. 관리자 도메인 API 수정(평일 업무)
   - 농구장 등록 거절 API
   - newCourt 목록 조회 API
2. 레포 옮기는 방법 알아놓기(평일 업무 -> 주말에 시행)
3. 플로라가 아이디어주신 레디스 공부해보기(주말 업무)
